src/fty-common/web/audit_log.cc

```diff
@@ -28,7 +28,7 @@
 
 #include "../../fty_common_classes.h"
 
-Ftylog AuditLogManager::_auditlog = Ftylog("www-audit", FTY_COMMON_LOGGING_DEFAULT_CFG);
+Ftylog AuditLogManager::_auditlog = Ftylog("www-audit", "/etc/fty/wwwlog.cfg");
```

Comment:
Wouldn't it be better to use #define instead of hardcoded string, as it might be applied to other places as well?

Reply:
this is the only place to be used, so not really

Reply:
What about https://github.com/42ity/fty-rest/pull/514 then? And possibly in some proprietary rest repos?